pkg/virt-handler/launcher-clients/launcher-clients_test.go

```diff
@@ -203,8 +203,9 @@ var _ = Describe("VirtualMachineInstance migration target", func() {
 
 					// Expect the client to reconnect and succeed despite server restarts
 					client.SetCustomTimeouts(1*time.Second, 1*time.Second, 3*time.Second)
```

Comment:
Maybe tweaking here would be enough too, but I think Eventually().Should(Succeed()) is the right way to handle this anyway.

/lgtm

Reply:
basically you are right, those are used on the SendK8sEvent
so i am not sure if we need to add Eventually or tweak them, as it exists already

wdyt ?